Framework/API/src/MultipleFileProperty.cpp

```diff
@@ -216,6 +229,12 @@ MultipleFileProperty::setValueAsSingleFile(const std::string &propValue) {
  */
 std::string
 MultipleFileProperty::setValueAsMultipleFiles(const std::string &propValue) {
+  // if value is unchanged use the cached version
+  if ((propValue == m_oldPropValue) && (!m_oldFoundValue.empty())) {
+    return PropertyWithValue<std::vector<std::vector<std::string>>>::setValue(
+        toString(m_oldFoundValue));
```

Comment:
Would it make more sense to use  `PropertyWithValue<TYPE>::operator=(const TYPE &)` and avoid additional conversions between strings and vectors?